processor/README.md

```diff
@@ -254,7 +262,60 @@ Refer to [config.yaml](batchprocessor/testdata/config.yaml) for detailed
 examples on using the processor.
 
 
-## <a name="queued_retry"></a>Queued Retry Processor
+## <a name="memory-limiter"></a>Memory Limiter Processor
+
+The memory_limiter processor is used to prevent out of memory situations on
+the collector. Given that the amount and type of data a collector processes is
+environment specific and resource utilization of the collector is also dependent
+on the configured processors, it is important to put checks in place regarding
+memory usage. The memory_limiter processor offers the follow safeguards:
+- Ability to configure a ballast, which allocates memory and provides stability
+to the heap. In summary, the ballast increases the base size of the heap so that
+GC triggers are delayed and the number of GC cycles over time is reduced.
```

Comment:
This seems to imply the ballast is configured by memory_limiter, which is not true. The ballast is configured via "mem-ballast-size-mib" command line option. The "ballast_size_mib" configuration option in memory_limiter does not create a ballast, it merely informs memory limiter that the ballast exists, which is necessary for correct operation. In the future we will be able to get rid of this config option from memory limiter (we will need to add ability for components to know the command line options).

Reply:
Good call, fixed